cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -548,14 +550,14 @@ writeStakeSnapshot (StakePoolKeyHash hk) qState =
 getPoolStake :: KeyHash Cardano.Ledger.Keys.StakePool crypto -> SnapShot crypto -> Integer
 getPoolStake hash ss = pStake
   where
-    Coin pStake = fold s
-    (Stake s) = poolStake hash (_delegations ss) (_stake ss)
+    Coin pStake = fold (Map.map Ledger.fromCompact $ VMap.toMap s)
+    Stake s = poolStake hash (_delegations ss) (_stake ss)
 
 -- | Sum the active stake from a snapshot
 getAllStake :: SnapShot crypto -> Integer
 getAllStake (SnapShot stake _ _) = activeStake
   where
-    Coin activeStake = fold . unStake $ stake
+    Coin activeStake = fold . Map.map Ledger.fromCompact . VMap.toMap $ unStake stake
```

Comment:
I suggest we merge https://github.com/input-output-hk/cardano-ledger/pull/2550/files and make use of `sumAllStake` there